config/opal_check_libnl_version.m4

```diff
@@ -0,0 +1,50 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2015      Research Organization for Information Science
+dnl                         and Technology (RIST). All rights reserved.
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+dnl OPAL_CHECK_LIBNL_INIT()
+dnl --------------------------------------------------------------------
+AC_DEFUN([OPAL_CHECK_LIBNL_INIT], [
+    opal_libnl_version=0
+    opal_libnl_libs=""
+    opal_libnl3_libs=""
+])
+
+dnl OPAL_CHECK_LIBNL_VERSION(lib)
+dnl --------------------------------------------------------------------
+AC_DEFUN([OPAL_CHECK_LIBNL_VERSION], [
+     AC_LANG_PUSH(C)
+     rm -f conftest_c.$ac_ext
+      cat > conftest_c.$ac_ext << EOF
+int main(int argc, char *argv[]) {
+    return 0;
+}
+EOF
+     OPAL_LOG_COMMAND([$CC -o conftest $CFLAGS $CPPFLAGS conftest_c.$ac_ext $LDFLAGS -l$1],
```

Comment:
It's possible, but I don't think it's likely -- this is a slightly different situation: we're linking in libraries which have already successfully been linked.  If it happens, though, we can adjust.

As for twice as long for this individual test, yes, this is true, but I'm not worry about it.  configure already takes a long time on a fast machine.  Also, 2 links instead of 1 is still a very short amount of time.